sal_interfaces/MTMount/MTMount_Telemetry.xml

```diff
@@ -6,291 +6,129 @@
-          <Units>deg_C</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Auxiliar_Azimuth_Cabinet_Internal_Temperature_1</EFDB_Name>
-          <Description>Auxiliar_Azimuth_Cabinet_Internal_Temperature_1</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>deg_C</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Auxiliar_Azimuth_Cabinet_Internal_Temperature_2</EFDB_Name>
-          <Description>Auxiliar_Azimuth_Cabinet_Internal_Temperature_2</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>deg_C</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Auxiliar_Azimuth_Cabinet_Setpoint_Temperature</EFDB_Name>
-          <Description>Auxiliar_Azimuth_Cabinet_Setpoint_Temperature</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>deg_C</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>timestamp</EFDB_Name>
-          <Description>timestamp</Description>
-          <Size_in_bytes>1</Size_in_bytes>
-          <IDL_Type>double</IDL_Type>
-          <Units>second</Units>
-          <Count>1</Count>
-      </item>
-</SALTelemetry>
-<SALTelemetry>
- <Subsystem>MTMount</Subsystem>
- <Version>3.2.0</Version>
- <Author>Ander Ansuategi</Author>
- <EFDB_Topic>MTMount_General</EFDB_Topic>
-     <item>
-          <EFDB_Name>AmbientTemperature</EFDB_Name>
-          <Description>AmbientTemperature</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>deg_C</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>timestamp</EFDB_Name>
-          <Description>timestamp</Description>
-          <Size_in_bytes>1</Size_in_bytes>
-          <IDL_Type>double</IDL_Type>
-          <Units>second</Units>
-          <Count>1</Count>
-      </item>
-</SALTelemetry>
-<SALTelemetry>
- <Subsystem>MTMount</Subsystem>
- <Version>3.2.0</Version>
- <Author>Ander Ansuategi</Author>
- <EFDB_Topic>MTMount_Azimuth_Cable_Wrap</EFDB_Topic>
+ <EFDB_Topic>MTMount_Camera_Cable_Wrap</EFDB_Topic>
      <item>
-          <EFDB_Name>ACW_Status</EFDB_Name>
-          <Description>ACW_Status</Description>
-          <Size_in_bytes>1</Size_in_bytes>
+          <EFDB_Name>CCW_Status</EFDB_Name>
+          <Description>CCW_Status</Description>
+          <IDL_Size>1</IDL_Size>
           <IDL_Type>string</IDL_Type>
-          <Units>unitless</Units>
+          <Units>   </Units>
           <Count>1</Count>
       </item>
      <item>
-          <EFDB_Name>ACW_Status_Drive_1</EFDB_Name>
-          <Description>ACW_Status_Drive_1</Description>
-          <Size_in_bytes>1</Size_in_bytes>
+          <EFDB_Name>CCW_Status_Drive_1</EFDB_Name>
+          <Description>CCW_Status_Drive_1</Description>
+          <IDL_Size>1</IDL_Size>
           <IDL_Type>string</IDL_Type>
-          <Units>unitless</Units>
+          <Units>   </Units>
```

Comment:
Units cannot be blank or just whitespace.  Please use `unitless` or `dimensionless`.